src/telemetry/ConfirmationViewTelemetryProps.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+export interface ConfirmationViewTelemetryProps {
```

Comment:
Why are these all only true? Shouldn't they be 'true' | 'false' ?

Reply:
I was following what Matt Fisher did for the azd telemetry props: https://github.com/microsoft/vscode-azurecontainerapps/blob/main/src/telemetry/AzdTelemetryProps.ts. 

My thoughts on this is we only really care if this event is true. I thought this would basically give us a count of how many users are doing these events but let me know if that thought process is incorrect.

Reply:
I guess we should add `isCopilotEvent` is as well?

Reply:
In this case these aren't optional, so I would either make them optional or allow 'false' **